scripts/ec2/ls.coffee

```diff
@@ -152,27 +160,30 @@ messages_from_ec2_instances = (instances) ->
     name = '[NoName]'
     for tag in instance.Tags when tag.Key is 'Name'
       name = tag.Value
+    description = ''
+    for tag in instance.Tags when tag.Key is 'Description'
+      description = tag.Value
 
     messages.push({
-      time: moment(instance.LaunchTime).format('YYYY-MM-DD HH:mm:ssZ')
+      time: moment(instance.LaunchTime).format('YYYY-MM-DD')
```

Comment:
Time will not be useful for most users; removing in favor of giving more space for description